pybossa/core.py

```diff
@@ -689,7 +689,9 @@ def get_wizard_steps(project=None):
             plugins=plugins,
             ldap_enabled=ldap_enabled,
             get_wizard_steps=get_wizard_steps,
-            server_type=app.config['SERVER_TYPE'])
+            server_type=app.config['SERVER_TYPE'],
+            favicon_dir=app.config['FAVICONS']
```

Comment:
I believe this change may not be needed. The HTML file can simply read the favicon filename as a string directly from the HTML. In this case, we can close this PR unmerged. Thanks.

Reply:
I confirmed with Deep that in order to access the FAVICON variable value in base.html we need to define it here.

Reply:
This still may not be required. Please see this [example](https://github.com/bloomberg/pybossa-default-theme/blob/main/templates/projects/update.html#L58) where a config variable is accessed from the HTML template. The same could be done for `FAVICON_FILENAME`. Please feel free to test in the code to verify. Thanks.